doc/source/parameters/cfd/boundary_conditions_multiphysics.rst

```diff
@@ -0,0 +1,55 @@
+==================================
+Multiphysics - Boundary Contitions
+==================================
+
+This subsection's purpose is defining the boundary conditions associated to multiphysic problems. 
+
+Heat Transfer
+^^^^^^^^^^^^^
+
+.. code-block:: text
+
+    subsection boundary conditions heat transfer
+    set number                  = 2
+        subsection bc 0
+            set type	        = temperature
+            set value	        = 0
+        end
+        subsection bc 1
+            set type		    = convection
+            set h 		    = 10
+            set Tinf	   	    = -13
+        end
+    end
+
+* ``number``: This is the number of boundary conditions of the problem. 
+
+* ``type`` : This is the type of boundary condition been imposed. At the moment, choices are 
+* ``temperature`` (Dirichlet BC), to impose a given temperature ``value`` at the boundary 
+* ``convection`` (Robin BC) for cooling/heating, depending on the environment temperature at the boundary ``Tinf``, with a given heat transfer coefficient ``h`` following Newton's law of cooling (and heating)
```

Comment:
I'm not able to do it on the browser, but you could insert a tab in front of the temperature and convection bullets, it will do as in https://lethe-cfd.github.io/lethe/parameters/cfd/physical_properties.html

Reply:
You mean as a second level for the bullet points?

Reply:
yes! :)